Changelog.md

```diff
@@ -1,5 +1,207 @@
+    * [Pull request #Merge 11 into 12](https://github.com/ignitionrobotics/sdformat/pull/Merge 11 into 12)
+
+1. Add ParserConfig flag for preserveFixedJoint
+    * [Pull request #815](https://github.com/ignitionrobotics/sdformat/pull/815)
+
+1. Fix parsing 'type' attibutes in plugins
+    * [Pull request #809](https://github.com/ignitionrobotics/sdformat/pull/809)
+
+1. Merge pull request #810 from scpeters/merge_11_to_12
+    * [Pull request #810 from scpeters/merge_11_to_12](https://github.com/ignitionrobotics/sdformat/pull/810 from scpeters/merge_11_to_12)
+
+1. Fix compiler warnings
+    * [Pull request #810](https://github.com/ignitionrobotics/sdformat/pull/810)
+
+1. Merge branch 'sdf11' into merge_11_to_12
+    * [Pull request #Merge branch 'sdf11' into merge_11_to_12](https://github.com/ignitionrobotics/sdformat/pull/Merge branch 'sdf11' into merge_11_to_12)
+
+1. Merge pull request #808 from scpeters/merge_10_11
+    * [Pull request #808 from scpeters/merge_10_11](https://github.com/ignitionrobotics/sdformat/pull/808 from scpeters/merge_10_11)
+
+1. Fix compiler warnings
+    * [Pull request #808](https://github.com/ignitionrobotics/sdformat/pull/808)
+
+1. Merge branch 'sdf10' into merge_10_11
+    * [Pull request #Merge branch 'sdf10' into merge_10_11](https://github.com/ignitionrobotics/sdformat/pull/Merge branch 'sdf10' into merge_10_11)
+
+1. Merge pull request #807 from scpeters/merge_9_to_11
+    * [Pull request #807 from scpeters/merge_9_to_11](https://github.com/ignitionrobotics/sdformat/pull/807 from scpeters/merge_9_to_11)
+
+1. sdf_custom: fix nested model expectations
+    * [Pull request #807](https://github.com/ignitionrobotics/sdformat/pull/807)
+
+1. Merge commit '6b53f954b8b74d0bdba33982d59c25da8b280ee4' into merge_9_to_11
+    * [Pull request #Merge commit '6b53f954b8b74d0bdba33982d59c25da8b280ee4' into merge_9_to_11](https://github.com/ignitionrobotics/sdformat/pull/Merge commit '6b53f954b8b74d0bdba33982d59c25da8b280ee4' into merge_9_to_11)
+
+1. Prepare for 10.7.0, final Dome release
+    * [Pull request #804](https://github.com/ignitionrobotics/sdformat/pull/804)
+
+1. Prepare for 10.7.0~pre2
+    * [Pull request #802](https://github.com/ignitionrobotics/sdformat/pull/802)
+
+1. Fix test compilation with USE_INTERNAL_URDF
+    * [Pull request #800](https://github.com/ignitionrobotics/sdformat/pull/800)
+
+1. Prepare for 10.7.0~pre1
+    * [Pull request #799](https://github.com/ignitionrobotics/sdformat/pull/799)
+
+1. Replace custom cmake code with ign-cmake2
+    * [Pull request #780](https://github.com/ignitionrobotics/sdformat/pull/780)
+
+1. 9 ➡️ 10
+    * [Pull request #795](https://github.com/ignitionrobotics/sdformat/pull/795)
+
+1. Support printing sdf poses in degrees and allow snapping to commonly used angles
+    * [Pull request #689](https://github.com/ignitionrobotics/sdformat/pull/689)
+
+1. 9 ➡️  10
+    * [Pull request #9 ➡️  10](https://github.com/ignitionrobotics/sdformat/pull/9 ➡️  10)
```

Comment:
The forward-ports can also be ommitted, because the PRs that were actually ported should already show up as separate items